src/index.html

```diff
@@ -1,17 +1,175 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="first">
+          <label for="in1">Surname:</label>
+          <input type="text"
+                 id="in1"
+                 autocomplete="off"
+                 name="surname"
+          >
+        </div>
```

Comment:
apply code style [recommendation](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) all over the page
```suggestion
        <div class="first">
          <label for="in1">Surname:</label>
          <input 
            type="text"
            id="in1"
            autocomplete="off"
            name="surname"
          >
        </div>
```